### PR TITLE
Update module's documentation and tasks

### PR DESCRIPTION
**Pull request by johlju**
#### Pull Request (PR) description

- Update the documentation and the tasks.
 
#### This Pull Request (PR) fixes the following issues

Closes https://github.com/gaelcolas/Sampler/issues/351

#### Task list

- [x] Added an entry to the change log under the Unreleased section of the file CHANGELOG.md.
      Entry should say what was changed and how that affects users (if applicable), and
      reference the issue being resolved (if applicable).
- [x] Documentation added/updated in README.md.
- [ ] Comment-based help added/updated.
- [ ] Localization strings added/updated in all localization files as appropriate.
- [ ] Examples appropriately added/updated.
- [x] Unit tests added/updated. See [DSC Community Testing Guidelines](https://dsccommunity.org/guidelines/testing-guidelines).
- [x] Integration tests added/updated (where possible). See [DSC Community Testing Guidelines](https://dsccommunity.org/guidelines/testing-guidelines).
- [x] New/changed code adheres to [DSC Community Style Guidelines](https://dsccommunity.org/styleguidelines).

---
This change is [Reviewable](https://reviewable.io/reviews/synedgy/sampler.azuredevopstasks/1)

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -146,6 +145,6 @@ TaskHeader: |
 GitHubConfig:
   GitHubFilesToAdd:
     - 'CHANGELOG.md'
-  GitHubConfigUserName: dscbot
-  GitHubConfigUserEmail: dsccommunity@outlook.com
+  #GitHubConfigUserName: dscbot
+  #GitHubConfigUserEmail: dsccommunity@outlook.com
```
Comment (johlju) [inline build.yaml:149]: @gaelcolas hat do you want me to use for these two? Same as Sampler, or do you want to set up another account for this (sending in the change log PR)?

**Comment by gaelcolas** [comment]
Lmk when ready btw 😉

**Comment by johlju** [comment]
Yeah. It is a bit more work to get it done. I will see if I have more time tomorrow. 

Need to get configuration options for the task into build.yaml, and then fix the tests.

**Comment by johlju** [comment]
@gaelcolas okay this is ready now!

**Review by gaelcolas — APPROVED** [review]
Thanks, that looks awesome!

**Pull request merged**